includes/class-sensei-course.php

```diff
@@ -222,7 +222,12 @@ public function can_access_course_content( $course_id, $user_id = null, $context
 	private static function add_course_access_permission_message( $message ) {
 		global $post;
 		if ( Sensei()->settings->get( 'access_permission' ) ) {
-			$message = apply_filters( 'sensei_couse_access_permission_message', $message, $post->ID );
+			$message = apply_filters_deprecated(
```

Comment:
When this filter is removed, should this method also be removed? And its usages?

Reply:
I think not. In the `output_course_enrolment_actions` method we have some usages to show some messages in the course, for example: _"or log in to view this course."_